h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -595,11 +595,11 @@ public TwoDimTable createScoringHistoryTable() {
 
     if (null == m) {
       Log.warn("Cannot create grid scoring history table; Model has been removed: " + k);
-      return ScoringInfo.createScoringHistoryTable(_scoring_infos, false, false, ModelCategory.Binomial, false);
+      return ScoringInfo.createScoringHistoryTable(_scoring_infos, false, false, ModelCategory.Binomial, false, false);
     }
 
     ScoringInfo scoring_info = _scoring_infos != null && _scoring_infos.length > 0 ? _scoring_infos[0] : null;
-    return ScoringInfo.createScoringHistoryTable(_scoring_infos, (scoring_info != null ? scoring_info.validation : false), (scoring_info != null ? scoring_info.cross_validation: false), m._output.getModelCategory(), (scoring_info != null ? scoring_info.is_autoencoder : false));
+    return ScoringInfo.createScoringHistoryTable(_scoring_infos, (scoring_info != null ? scoring_info.validation : false), (scoring_info != null ? scoring_info.cross_validation: false), m._output.getModelCategory(), (scoring_info != null ? scoring_info.is_autoencoder : false), false);
```

Comment:
Shouldn't the `false` be something like `m._parms.hasCustomMetric()` here?
I think we can provide the custom metric to the grid (in python by putting it to the model estimator constructor that is passed to the grid constructor) so it seems to me that we could use it here as well.

Reply:
Good point, @tomasfryda. For some reason, `m._parms.hasCustomMetric()` failed (maybe Grid does not have parameters?), so I just left this parameter with false in this PR, and I would like to solve this problem for Grid in a separate PR. WDYT?

Reply:
Weird. I'm on a different branch but IDE auto completes `m._parms._custom_metric_func` so I'd expect it would work.  And `m` seems to be a model. So I don't see a reason why it shouldn't work. 
![Screen Shot 2023-06-22 at 15 11 18](https://github.com/h2oai/h2o-3/assets/61695433/18ff1582-42b8-49eb-a0ca-22a130fce9f1)

But feel free to do it in a separate PR if you think it will take more than changing this line. (And if you find out why it failed, I'd be interested to know that reason. I must be missing something 🤔 )

Reply:
Thanks for trying it. I will look into the code again and try to fix it. I am probably wrong if it works for you. :)

Reply:
It was my bad; I fixed it now. Thanks @tomasfryda!